source/unified-test-format/unified-test-format.md

```diff
@@ -1416,6 +1429,53 @@ the resulting change stream might be saved with [operation.saveResultAsEntity](#
 Test runners MUST NOT iterate the change stream when executing this operation and test files SHOULD NOT specify
 [operation.expectResult](#operation_expectResult) for this operation.
 
+#### clientBulkWrite
```

Comment:
This should just be `bulkWrite`, as we're already under a "Client Operations" heading. If you'd like another link to it you can add something like the following:

```
<span id="client_bulkWrite" />
```

That's similar to what was done for `createChangeStream` above (see: https://github.com/mongodb/specifications/commit/68da5c5c9af9d87677ed90d30f8ad9ca3ea7e798).

Existing links to `#bulkwrite` can be changed to `#collection_bulkWrite` if you're willing to add a `<span id="collection_bulkWrite" />` above its heading as well.

Reply:
I intentionally named the UTR operation for the new bulk write "clientBulkWrite" to avoid conflicts between collection bulk write and client bulk write. This was useful in Rust, and I imagine it will be for other drivers that convert test JSON/YML into native types since the two APIs use different types. I used the "clientBulkWrite" name here for consistency with the unified tests.

Note: we do use the same name for collection- and database-level aggregate in the unified test runner, which was the only precedent I could think of for this situation. However, these operations have the same arguments, so the same deserialization concerns don't apply.

Reply:
I didn't realize `clientBulkWrite` was also being used in tests. Operation names have historically matched the defining spec, excluding custom operations introduced by the test runner (e.g. `createChangeStream`, `createFindCursor`).

If this is required, please make a note of why this naming deviation was done somewhere in the spec. I'm more concerned about this opening the door for future deviations.

Reply:
Gotcha, I added a blurb to the "Entity Test Operations" section to clarify that a) test files should use the same names as those defined in specs and b) why we're deviating here.